Base/usr/share/man/man1/Applications/3DFileViewer.md

```diff
@@ -16,16 +16,46 @@ $ 3DFileViewer
 
 It currently supports opening the Wavefront OBJ file format (`.obj`).
 
-### Features
+## Features
 
-By default 3D File Viewer will open with a rotating [Utah teapot](https://en.wikipedia.org/wiki/Utah_teapot), a standard 3D test model.
+By default, 3D File Viewer opens with a rotating [Utah teapot](https://en.wikipedia.org/wiki/Utah_teapot), a standard 3D test model. Open files via *File → Open* (`Ctrl+O`) or dragging and dropping files into the application. Examples files can be found in `Documents/3D Models`.
 
-Rotate the model by grabbing it with the cursor. Zoom-in and out with the mouse wheel.
+Rotate the model by grabbing it with the cursor, and zoom-in and out with the mouse wheel.
 
-The rotation axis and speed can be adjusted or turned off in the `View` menu.
+Settings persist whilst the application is open with different files, but does not persist between sessions.
 
-The model's texture can be adjusted or turned off in the `Texture` menu.
+### View Options
 
-View in Fullscreen by pressing `F11`.
+* View in Fullscreen mode by pressing `F11` and press `Esc` to return to windowed mode.
+* **Rotation Axis** - Enable or disable rotation on the right-handed coordinate system.
+    * **X** rotates upwards and back, away from the viewer
+    * **Y** rotates right
+    * **Z** rotates clockwise
+* **Rotation Speed** - Set to slow, normal, fast or disabled.
+* **Show Frame Rate** - Display the FPS (frames per second) in the top-right corner, along with frame time values indicating how long it took to render a single frame.
 
-Drag and drop files into 3D File Viewer to open them.
\ No newline at end of file
+### Texture Options
+
+Loading a texture requires a Bitmap image file (`.bmp`) with the model's name in the same folder. 3D File Viewer will attempt to load this by default.
+
+* **Enable Texture** - On by default, disable for a blank (gray) model.
+* **Wrap S or T** - Controls how a texture is applied horizontally (S) or vertically (T):
+    * **Repeat** - Tiles the texture repeatedly along the axis.
+    * **Mirrored** - Tiles and mirrors the texture along the axis.
+    * **Clamp** - Stretches or clamps the texture edges instead of repeating it.
+* **Scale** - Adjusts texture size (e.g. `0.5` reduces by half, `2x` doubles the size).
+* **Mag Filter** - Affects how model appears up-close:
+    * **Nearest** - Pixelated for a blocky look.
+    * **Linear** - Smoother by blending nearby texels (tiny pictures on a 3D object's surface).
```

Comment:
I would just reference the commonly known image scaling algorithms here; i.e. nearest neighbour and linear interpolation.